docs/RelayAPI.md

```diff
@@ -99,14 +99,20 @@ curl https://relay0.testnet.trustlines.network/api/v1/networks/0xC0B33D88C704455
 |decimals|int|Decimals specified in currency network|
 |name|string|Full name of the currency network|
 |numUsers|int|Total number of users in currency network|
+|defaultInterests|int|The default interest interest rate for every user in the network in 0.001% per year|
+|customInterests|bool|Whether custom interest rate can be set by users|
+|safeInterestRippling|bool|Whether the safe rippling strategy is set|
 #### Example Response
 ```json
 {
   "abbreviation": "HOU",
   "address": "0xC0B33D88C704455075a0724AA167a286da778DDE",
   "decimals": 2,
   "name": "Hours",
-  "numUsers": 3
+  "numUsers": 3,
+  "defaultInterests": 1000,
+  "customInterests": false,
+  "safeInterestRippling": false
 }
```

Comment:
We could, but the interestRateDecimals is fixed so that interests are expressed in 0.01% per year. It cannot be modified, so does it make sense to have it in the endpoint?

Reply:
Ahh ok... but are there plans in the future to make it configurable? I think @berndbohmeier mentioned it at some point. But I would just include them for now so we could change it later. Will also ping Bernd to get some input from him about this.

Reply:
I can include that, so that we can change it later.